src/Uno.UI/UI/Xaml/Controls/Image/Image.Android.cs

```diff
@@ -36,11 +37,13 @@ private Windows.Foundation.Size SourceImageSize
 		/// <param name="isLogicalPixels">indicates that the size of the image source is in logical pixels (this is the case when the source is an URI)</param>
 		internal void UpdateSourceImageSize(Windows.Foundation.Size size, bool isLogicalPixels = false)
 		{
-			if (_sourceImageSize == size)
+			if (_sourceImageSize == size && _isResized)
```

Comment:
Why do you need to detect resizing here ?

Reply:
It's unclear what `_isResized` means.